views/confirmation.html

```diff
@@ -61,9 +67,15 @@ <h3 class="govuk-heading-s">What happens if the application is rejected</h3>
         <br/>
       {% endif %}
 
-      <p class="govuk-body">
-        <a class="govuk-link" href={{ "https://www.smartsurvey.co.uk/s/oe-update-conf/" if isUpdate else "https://www.smartsurvey.co.uk/s/roe-confirmation/" }}>What did you think of this service?</a> (takes 30 seconds)
-      </p>
+      {% if isUpdate %}
+        <p class="govuk-body">
+          <a class="govuk-link" href={{ "https://www.smartsurvey.co.uk/s/oe-remove-confirmation/" if (journey == "remove") else "https://www.smartsurvey.co.uk/s/oe-update-conf/" }}>What did you think of this service?</a> (takes 30 seconds)
+        </p>
+      {% else %}
+        <p class="govuk-body">
```

Comment:
looks like the else part of this is new but will also be shown if registration journey , is that correct?

Reply:
No, tested all scenarios (reg, update and remove) and screen shows correctly.